blog-homepage/src/blogUI.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './blogUI.css';
+import missedArticles from './missed-articles.json';
+import yourArticles from './your-articles.json';
+import Article from './Article.js';
+
+class BlogUI extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    createCards = (articles) => articles.map(article => <Article article={article} />);
```

Comment:
Don't forget your index as a key!
```suggestion
    createCards = (articles) => articles.map((article,index) => <Article article={article} key={index} />);
```